Python/Product/Cookiecutter/ViewModel/CookiecutterViewModel.cs

```diff
@@ -569,11 +584,14 @@ class CookiecutterViewModel : INotifyPropertyChanged {
             IsCreatingError = false;
             IsCreatingSuccess = false;
 
+            var selection = SelectedTemplate;
+            Debug.Assert(selection != null);
```

Comment:
We should do something here if `selection` is null - either raise a more useful exception or abort the operation with an error for the user.